pkg/build/builder/sti.go

```diff
@@ -227,6 +227,13 @@ func (s *S2IBuilder) Build() error {
 		config.RuntimeImage = runtimeImageName
 		config.RuntimeAuthentication, _ = dockercfg.NewHelper().GetDockerAuth(runtimeImageName, dockercfg.PullAuthType)
 		config.RuntimeArtifacts = copyToVolumeList(s.build.Spec.Strategy.SourceStrategy.RuntimeArtifacts)
+		if s.build.Spec.Strategy.SourceStrategy.ForcePull {
+			glog.V(4).Infof("With force pull true, setting runtime image pull policies to %s", s2iapi.PullAlways)
+			config.RuntimeImagePullPolicy = s2iapi.PullAlways
+		} else {
+			glog.V(4).Infof("With force pull false, setting runtime image pull policies to %s", s2iapi.PullIfNotPresent)
+			config.RuntimeImagePullPolicy = s2iapi.PullIfNotPresent
+		}
```

Comment:
@php-coder s.build.Spec.Strategy.SourceStrategy.ForcePull sometime will be nil , if we don't set this in the bc, so I think twice of the log is acceptable.